source/docs/software/basic-programming/java-units.rst

```diff
@@ -1,58 +1,107 @@
++--------------------------------------------------------+------------------------------------------------------------------------------------------------------------------------+
+| ``Measure<Power>``                                     | ``Watts``, ``Milliwatts``, ``Horsepower``                                                                              |
++--------------------------------------------------------+------------------------------------------------------------------------------------------------------------------------+
+| ``Measure<Temperature>``                               | ``Kelvin``, ``Celsius``, ``Fahrenheit``                                                                                |
++--------------------------------------------------------+------------------------------------------------------------------------------------------------------------------------+
+| ``Measure<Per<Voltage, Velocity<Distance>>>``          | ``VoltsPerMeterPerSecond``                                                                                             |
++--------------------------------------------------------+------------------------------------------------------------------------------------------------------------------------+
+| ``Measure<Per<Voltage, Velocity<Velocity<Distance>>>`` | ``VoltsPerMeterPerSecondSquared``                                                                                      |
++--------------------------------------------------------+------------------------------------------------------------------------------------------------------------------------+
+| ``Measure<Per<Voltage, Velocity<Angle>>>``             | ``VoltsPerRadianPerSecond``                                                                                            |
++--------------------------------------------------------+------------------------------------------------------------------------------------------------------------------------+
+| ``Measure<Per<Voltage, Velocity<Velocity<Angle>>>``    | ``VoltsPerRadianPerSecondSquared``                                                                                     |
++--------------------------------------------------------+------------------------------------------------------------------------------------------------------------------------+
+
+Creating Measures
+~~~~~~~~~~~~~~~~~
+
+The ``Measure`` class is a generic type that represents a physical quantity with its corresponding unit of measurement. It provides a consistent and type-safe way to handle different types of measurements, such as distance, angle, velocity, and more. To create a ``Measure`` object, you call the ``Unit.of`` method on the appropriate unit object. For example, to create a ``Measure<Distance>`` object representing a distance of 6 inches, you would write:
```

Comment:
The way you've phrased this I think it would be easy to mistakenly assume that `Measure` is parameterized by the unit. I think you should explicitly say that `Measure` is parameterized by the dimension rather than a particular unit, since that's half the point of the library.

Maybe something like:
> The measure class is a generic type representing a physical quantity. It tracks the type of measurement (distance, angle, mass, etc.), but abstracts away the particular unit of measure (e.g. meter vs. inch) unless you explicitly convert to a particular unit.

Reply:
I like that, I'll add the bit about abstracting the unit of measure. It's under the heading "Creating objects" so I'm going to leave out "unless you explicitly convert to a particular unit"